### PR TITLE
build(workflow): update dependency on event

### PR DESCRIPTION
**Pull request by maloun96**

### CODE REVIEW

File: .github/workflows/update-dependencies.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+jobs:
+  update-webex-component-adapter-interfaces:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'update-webex-component-adapter-interfaces'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
+      - run: |
+          wanted=$(npm outdated @webex/component-adapter-interfaces --json | grep "wanted")
```
Comment (cipak) [inline .github/workflows/update-dependencies.yml:13]: maybe a simpler version
```sh
npm outdated @webex/components --json | grep wanted | cut -f4 -d\"
1.62.0
```

`npm outdated` has also a `--parseable` option but it separates tokens by colons (:), and on windows the same character is used for the drive name (eg: `C:`)

File: .github/workflows/update-dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          echo ${wanted}
+
+          current=$(npm outdated @webex/component-adapter-interfaces --json | grep "current")
+          IFS='"' read -ra current <<< "$current"
+          current=${current[3]}
+          echo ${current}
+
+          if [ "$current" != "$wanted" ]; then
+            echo "Not Equal"
+            npm install @webex/component-adapter-interfaces@${wanted} --save
```
Comment (cipak) [inline .github/workflows/update-dependencies.yml:25]: I guess `--save` is implicit now

File: .github/workflows/update-dependencies.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
+      - run: |
+          npm install @webex/component-adapter-interfaces --save
+          if [ -n "$(git status --porcelain)" ]; then
+            git config user.name "GitHub Actions Bot"
+            git config user.email "<>"
+            git add .
+            git commit -m 'fix(@webex/component-adapter-interfaces): update to new version'
```
Comment (cipak) [inline .github/workflows/update-dependencies.yml:18]: @lalli-flores I would prefer dependency updates to use `build()`, as they should not yet use the features in the new dependency version (eg: a new field in an adapter interface). It's the subsequent commits that start using those features, and it's those commits that should generate new versions on the current repository.

Comment (cipak) [reply .github/workflows/update-dependencies.yml:18]: I would prefer the same commit message we are currently using:
`build(package): update component adapter interfaces to v1.2.3`

File: .github/workflows/update-dependencies.yml (hunk 2 above)
Comment (cipak) [reply .github/workflows/update-dependencies.yml]: so you can remove it

File: .github/workflows/update-dependencies.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    if: github.event.action == 'component-adapter-interfaces:version-changed'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
+      - run: |
+          npm install @webex/component-adapter-interfaces
+          if [ -n "$(git status --porcelain)" ]; then
+            git config user.name "GitHub Actions Bot"
+            git config user.email "<>"
```
Comment (cipak) [inline .github/workflows/update-dependencies.yml:16]: `41898282+github-actions[bot]@users.noreply.github.com`

taken from here: https://github.community/t/github-actions-bot-email-address/17204/6

File: .github/workflows/update-dependencies.yml (hunk 3 above)
Comment (lalli-flores) [reply .github/workflows/update-dependencies.yml]: yeah, I'm good with your commit message suggstion

File: .github/workflows/update-dependencies.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+on: [repository_dispatch]
+
+jobs:
+  update-webex-component-adapter-interfaces:
+    runs-on: ubuntu-latest
+    if: github.event.action == 'component-adapter-interfaces:version-changed'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2-beta
```
Comment (lalli-flores) [inline .github/workflows/update-dependencies.yml:11]: Sorry, I just noticed this, but why are we using the beta version here?

**Review by lalli-flores — APPROVED** [review]

**Pull request merged**